src/bindings/python/tests/test_ngraph/test_create_op.py

```diff
@@ -109,7 +109,7 @@ def test_ctc_greedy_decoder(dtype):
                              (np.float64, np.int64, "i64", "i32", False, False),
                              (np.float64, np.int64, "i32", "i64", False, False),
                              (np.float64, np.int64, "i64", "i64", False, False)
-                         ],)
+                         ], )
```

Comment:
What happened here?

Reply:
I applied PyCharm automatic formatting. May be we need to remove this comma at all, how do you think?